alchemiscale/tests/integration/test_cli.py

```diff
@@ -136,18 +142,92 @@ def test_compute_api(n4js, s3os):
     ]
     jwt_opts = []  # leaving empty, we have default behavior for these
 
+    return host, port, (command + api_opts + db_opts + s3_opts + jwt_opts)
+
+
+@pytest.fixture
+def compute_service_config(compute_api_args):
+    host, port, _ = compute_api_args
+
+    config = {
```

Comment:
If this YAML is required perhaps in example sometwhere in `devtools` is a good idea?

Reply:
Good call, will add an example file. I think this may get better with #107 as well.